client/src/intro-exit/Surveys/quality_control.jsx

```diff
@@ -1,26 +1,39 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useState } from "react";
 import SurveyWrapper from "../../components/SurveyWrapper";
 import surveyJSON from './quality_control.json';
 import { usePlayer, useGame } from "@empirica/player";
+import { useCallback } from "react";
 
 
 
 export default function quality_control({ next }) {
     const player = usePlayer();
     const game = useGame();
-    
+    const [displayPayment, setDisplayPayment] = useState("calculating...");
     useEffect(() => {
+        console.log("isPaidTime before" + player.get("isPaidTime"))
         player.set("isPaidTime", false); //stop paying participant when they get to this screen (so we can compute the time)
+        console.log("isPaidTime after" + player.get("isPaidTime"))
         console.log("QC Exit. Played for " + player.get("activeMinutes") + " minutes, earned $" + player.get("dollarsOwed"))
     }, [])
+
+    
     const dollarsOwed = player.get("dollarsOwed");
+    useEffect(() => {
```

Comment:
We should double check that this doesn't get called at render, but only when the dollarsOwed changes. Now that I think about it, when dollarsOwed is set from player.get, that might be enough to trigger this...
This might be the pattern we want: https://medium.com/swlh/prevent-useeffects-callback-firing-during-initial-render-the-armchair-critic-f71bc0e03536